src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -157,6 +161,48 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
         return Promise.resolve(trialJobOutputUrl);
     }
     
+    private generatePortScript(): string {
+        let portScript = '';
+        if(!this.frameworkcontrollerTrialConfig) {
+            throw new Error('frameworkcontroller trial config is not initialized');
+        }
+
+        for(let index in this.frameworkcontrollerTrialConfig.taskRoles) {
+            portScript += 
+            `${this.frameworkcontrollerTrialConfig.taskRoles[index].name}_port=${this.frameworkcontrollerContainerPortMap.get(this.frameworkcontrollerTrialConfig.taskRoles[index].name)} `;
```

Comment:
I suggest to use abbreviation for the term of 'frameworkcontroller', like 'fc', so we can dramatically reduce the length of variable name.